canonical-web-and-design/snapcraft.io
# Update channel map

## Done
- Update canonicalwebteam.store-api and replaced ["_embedded"]["clickindex:package"] code
- Update channel map to show "released-at" date instead of "created-at"

## How to QA
- Visit the demo in your browser, link commented below
  - Or you can run the project locally using the [dotrun](https://snapcraft.io/dotrun) snap with `$ dotrun` and view it in your web browser at: http://0.0.0.0:8004/
  - If your demo doesn't work, read more about the [demoservice](https://discourse.ubuntu.com/t/demoservice/16862)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check all the consumer part works fine

## Issue / Card
Fixes https://github.com/canonical-web-and-design/snap-squad/issues/1759
